C4SGSvc/src/main/java/org/c4sg/entity/User.java

```diff
@@ -54,6 +61,99 @@
  
     @Column(name = "location", columnDefinition = "point")
     private Point location;
+    
+    @Column(name = "introduction",nullable = true)
+    private String introduction;
+    
+    @Column(name = "linked_inurl",nullable = true)
+    private String linked_inurl;
+    @Column(name = "personal_web_site",nullable = true)
+    private String personal_web_site;
+    @Column(name = "resume",nullable = true)
+    private String resume;
+    @Column(name = "skill1",nullable = true)
+    private String skill1;
+    @Column(name = "skill2",nullable = true)
+    private String skill2;
+    @Column(name = "skill3",nullable = true)
+    private String skill3;
+    @Column(name = "skill4",nullable = true)
+    private String skill4;
+    public String getIntroduction() {
+		return introduction;
+	}
+
+	public void setIntroduction(String introduction) {
+		this.introduction = introduction;
+	}
+
+	public String getLinked_inurl() {
+		return linked_inurl;
+	}
+
+	public void setLinked_inurl(String linked_inurl) {
+		this.linked_inurl = linked_inurl;
+	}
+
+	public String getPersonal_web_site() {
+		return personal_web_site;
+	}
+
+	public void setPersonal_web_site(String personal_web_site) {
+		this.personal_web_site = personal_web_site;
+	}
+
+	public String getResume() {
+		return resume;
+	}
+
+	public void setResume(String resume) {
+		this.resume = resume;
+	}
+
+	public String getSkill1() {
+		return skill1;
+	}
+
+	public void setSkill1(String skill1) {
+		this.skill1 = skill1;
+	}
+
+	public String getSkill2() {
+		return skill2;
+	}
+
+	public void setSkill2(String skill2) {
+		this.skill2 = skill2;
+	}
+
+	public String getSkill3() {
+		return skill3;
+	}
+
+	public void setSkill3(String skill3) {
+		this.skill3 = skill3;
+	}
+
+	public String getSkill4() {
+		return skill4;
+	}
+
+	public void setSkill4(String skill4) {
+		this.skill4 = skill4;
+	}
+
+	public String getSkill5() {
+		return skill5;
+	}
+
+	public void setSkill5(String skill5) {
+		this.skill5 = skill5;
+	}
+
+	@Column(name = "skill5",nullable = true)
+    private String skill5;
```

Comment:
The reason they are not in a linkage table is because there are a max of 5 skills allowed only. So it is convenient to place in user table directly.